### DISCUSSION TITLE
Replacing <strong> with <b> and <em> with <i>

### DISCUSSION THREAD

**Discussion by informatikTirol** [sibiraj-s/ngx-editor#382] [Q&A] [answered]
HI, 
`is there a way of replacing the strong label generated around the text when clicking into the "B" (Bold) button in the editor, by the <b> tag. Same way, for italic uses <em> instead of <i>?`

THX

**Comment by sibiraj-s** [comment, marked as answer]
You will need to create a custom schema and use it. For example using `i` for italics instead of `em`.

```js
import { nodes, marks } from 'ngx-editor';
import { Schema, DOMOutputSpec, MarkSpec } from 'prosemirror-model';

const em: MarkSpec = {
  parseDOM: [{ tag: 'i' }, { tag: 'em' }, { style: 'font-style=italic' }],
  toDOM(): DOMOutputSpec {
    return ['i', 0]; // <----- this line is changed. old `return ["em", 0];`
  },
};

const schema = new Schema({
  nodes,
  marks: {
    ...marks,
    em,
  },
});

const editor = new Editor({
  schema,
});
```

Refer this for all existing schema configuration https://github.com/sibiraj-s/ngx-editor/blob/master/projects/ngx-editor/schema/marks.ts

Also this for how to use custom schemas. https://github.com/sibiraj-s/ngx-editor/blob/master/projects/demo/src/app/schema.ts

**Reply by informatikTirol** [reply]
Hey @sibiraj-s great, it works. Thank you!

[a comment by an automated account is not shown]
